students/kimjy/westagram/users/views.py

```diff
@@ -44,3 +46,32 @@ def post(self, request):
         except IntegrityError as e:
             return JsonResponse({"message":str(e)}, status=409)
 
+class SignInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            
+            signin_key_list=["email", "password"]
+            CheckItem.check_keys_in_body(data, signin_key_list)
+
+            email    = data.get("email")
+            password = data.get("password")
+
+            user = User.objects.filter(email_address=email)
+            
+            if not user.exists():
+                raise EmptyResultSet("INVALID_USER")
+
+            if user[0].password != password:
+                raise PermissionDenied("INVALID_USER")
+
+            return JsonResponse({"message":"SUCCESS"}, status=200)
+
+        except KeyError as e:
+            return JsonResponse({"message":str(e)}, status=400)
```

Comment:
```suggestion
            return JsonResponse({"message":str(e.message)}, status=400)
```
e만 메시지를 보내게 되면 서버에서 나는 에러의 전문을 보내지 않나요??
그러면 보안상 문제가 될 수 있습니다.
input으로 넣어준 message만 갈 수 있도록 수정해 주세요.

Reply:
네, 확실히 e.message만 보내야지 에러의 전문이 나오지 않을 것 같습니다.
피드백주신 대로 코드 수정하였습니다.